docs/advanced/hooks.rst

```diff
@@ -122,4 +123,4 @@ A ``post_gen_project`` hook can conditionally control files and directories. The
     for path in REMOVE_PATHS:
         path = path.strip()
         if path and os.path.exists(path):
-            os.unlink(path) if os.path.isfile(path) else os.rmdir(path)
+            os.unlink(path) if os.path.isfile(path) else shutil.rmtree(path, ignore_errors=True)
```

Comment:
`path` is exclusively `requirements.txt` or `pyproject.toml`. In context, these are understood to _always_ be Python packaging files, and will _never_ be directories.

In other words, there should be no call to `os.rmdir()`, nor to `shutil.rmtree()`. This code should instead simply be:

```python
if path and os.path.isfile(path):
    os.unlink(path)
```